watertap/property_models/unit_specific/activated_sludge/asm3_reactions.py

```diff
@@ -0,0 +1,1013 @@
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        mu_H = self.params.mu_H["20C"] * pyo.exp(
+            theta_T_mu_H
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        b_H_O2 = self.params.b_H_O2["20C"] * pyo.exp(
+            theta_T_b_H_O2
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        b_H_NOX = self.params.b_H_NOX["20C"] * pyo.exp(
+            theta_T_b_H_NOX
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        b_STO_O2 = self.params.b_STO_O2["20C"] * pyo.exp(
+            theta_T_b_STO_O2
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        b_STO_NOX = self.params.b_STO_NOX["20C"] * pyo.exp(
+            theta_T_b_STO_NOX
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        mu_A = self.params.mu_A["20C"] * pyo.exp(
+            theta_T_mu_A
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        b_A_O2 = self.params.b_A_O2["20C"] * pyo.exp(
+            theta_T_b_A_O2
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
+        b_A_NOX = self.params.b_A_NOX["20C"] * pyo.exp(
+            theta_T_b_A_NOX
+            * (
+                pyo.units.convert(
+                    self.state_ref.temperature / pyo.units.K,
+                    to_units=pyo.units.dimensionless,
+                )
+                - (self.params.ref_temp_2 + 273.15)
+            )
+        )
```

Comment:
All of these can be (1) stored as Parameters/Expressions on the model for the user to trace back to and check the value of if desired (any reason why we think this wouldn't be desired?), and (2) we could make two functions to consolidate code (and reduce margin for error).